src/index.js

```diff
@@ -1,11 +1,45 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
- 
-  //return element;
-}
+const list = document.getElementById('list');
+const tasks = [
+  {
+    description: 'Clean my room.',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Eat dinner.',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'Get a developer job.',
+    completed: false,
+    index: 2,
+  },
+];
 
-component();
-//document.body.appendChild(component());
+const sortTask = (arr) => {
+  arr.sort((a, b) => ((a.index > b.index) ? 1 : -1));
+};
 
+const createTask = (tasks) => {
+  tasks.forEach((task) => {
```

Comment:
- [ ] I couldn't see if the project show the elements because there's no a `package.json` file. I invite you to add it to see if the project has the elements. 🐶